src/index.js

```diff
@@ -0,0 +1,39 @@
+import './style.css';
+
+class Tasks {
+  constructor() {
+    this.tasks = [];
```

Comment:
Great job we are just missing a simple thing
- [ ] Let's make sure we have simple examples of tasks that we dynamically render when the page loads as per project requirements 
![image](https://user-images.githubusercontent.com/87689547/159685125-6920ae26-78c1-409f-bc34-27557a4acf81.png)